ci-operator/step-registry/openshift/e2e/openstack/csi/openshift-e2e-openstack-csi-workflow.yaml

```diff
@@ -10,9 +10,6 @@ workflow:
     - ref: storage-pv-check
     - chain: ipi-openstack-post
     env:
-      OPENSTACK_EXTERNAL_NETWORK: "external"
-      OS_CLOUD: "openstack-cloud"
```

Comment:
Is it OK to remove the default value for OS_CLOUD? IIUC this workflow is not yet used, but who knows.

Reply:
Every step that uses `OS_CLOUD` declares (or should declare) it with default `openstack` (and not `openstack-cloud`). The workflows do not need to pass it, although they can. I don't see a reason for doing that, but they can.